packages/location-management/src/components/LocationForm/index.tsx

```diff
@@ -291,16 +289,6 @@ const LocationForm = (props: LocationFormProps) => {
             ></Radio.Group>
           </FormItem>
 
-          <FormItem
-            id="type"
-            rules={validationRules.type}
-            hidden={isHidden('type')}
-            name="type"
-            label={TYPE_LABEL}
-          >
-            <Input disabled={disabled.includes('type')} placeholder={SELECT_TYPE_LABEL} />
-          </FormItem>
-
```

Comment:
@dubdabasoduba do we want to remove this field, or hide it? cc @ashfahan

Reply:
as far as i know of the purpose of this was to remove this field and make it hard code to "feature" value.